packages/ai-jsx/src/core/prompts.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Basic prompts for LLMs.
+ * This file attemps to have suggestions to guide the thinking of inexperienced users
+ * to use good practices when using LLMs.
+ *
+ * Prompting is still a developing field. There are many different ways to prompt an LLM.
+ * This file attempts to provide some good defaults as a starting point.
+ */
```

Comment:
This is good context, but you should generally assume people won't see it. The parts people will see are the intellisense comments on the exported API members. So if there's critical context, it should live there.